train.py

```diff
@@ -146,12 +146,17 @@ def load_reader(pos: str):
 train_loader = data_loader.get_train_loader(batch_size=config["batch_size"], num_workers=0)
 valid_loader = data_loader.get_validation_loader(batch_size=config["batch_size"], num_workers=0)
 
+print("INFO: Training Dataset Size: {}, Minibatch Count: {}, Batch Size: {}".format(len(train_loader.dataset), len(train_loader), config["batch_size"]))
+print("INFO: Validation Dataset Size: {}, Minibatch Count: {}, Batch Size: {}".format(len(valid_loader.dataset), len(valid_loader), config["batch_size"]))
```

Comment:
Do you think it would be better to just add this information (dataset size, batch amount) to the config, so it's logged in wandb?

Reply:
Oh OK.